### PR TITLE
RFP: sarahos by Grandhelsman team

### PR DESCRIPTION
**Pull request by IPFS-grandhelmsman**
Lightweight distributed hardware management system for ipfs and filecoin.

### CODE REVIEW

File: rfp-proposals/rfp-community-proposal-GrandhelmsmanDocumentary.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+- Food and beverage plan: not yet.
+- Marketing and promoting party planning:
+- Plans to share records and post-event information:
+- Any follow-up activities you plan to organize:
+
+## Value
+
+What is the value of this event for the Filecoin ecosystem and community?
+
+Through this event, IPFS technology is gaining popularity among Chinese students, Guiding students to explore IPFS technology independently. In addition, the most innovative students can realize their inspiration by creating IPFS technology application.
```
Comment (pooja) [inline rfp-proposals/rfp-community-proposal-GrandhelmsmanDocumentary.md]: Thanks for the proposal! Unfortunately, we're only looking for proposals for Filecoin right now, not IPFS. You could reach out to the IPFS team in order to share this proposal with them!

File: rfp-proposals/rfp-proposal-sarahos.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+**Proposer:** `IPFS-grandhelmsman`
+
+**Do you agree to open source all work you do on behalf of this RFP and dual-license under MIT and APACHE2 licenses?:** "Yes"
+
+# Project Description
+
+Problem: if a developer wants to run and manage multiple ipfs-based projects on multiple servers, he should first solve the operation and maintenance of hardware nodes, including the reading of hardware running status information, alarm, etc, and next step is the projects of batch management.
+
+Solution: under our system, developers only need to add and optimize the projects on the platform we provide. Refer to the Sarah OS product documentation，there are a more detailed description.
```
Comment (pooja) [inline rfp-proposals/rfp-proposal-sarahos.md]: Could you share the link to Sarah OS product documentation? The product links don't seem to include documentation.

Comment (IPFS-grandhelmsman) [reply rfp-proposals/rfp-proposal-sarahos.md]: https://github.com/IPFS-grandhelmsman/sarah-doc

**Comment by pooja** [comment]
We've finished the wave 1 grant selection process. Can you please email devgrants@filecoin.org so we can share our decision and next steps, if any? In your email, please include a link to your proposal PR so we know who we're talking to!

Thanks,
Filecoin DevGrants team

**Comment by pooja** [comment]
Greetings,

Thank you for your application to the Filecoin DevGrants program. We appreciate your proposal!

You were not selected for a grant in Wave 1, but we hope that you consider applying again for a future DevGrant in a later wave.

We selected grants based on:
- what makes sense to build now based on where the Filecoin node implementations are
- the likelihood of your team being able to work independently of the Filecoin engineering teams working on the core client implementations
- degree of refactoring your proposal may require as the software clients evolve
- the integrity of the proposal and near-term value to the Filecoin ecosystem
- your commitment to the ecosystem and desire to maintain your project longer-term

As the Filecoin implementations advance towards Testnet and Mainnet, we hope you continue to consider new proposals for projects. We would love to see more applications from you in the future.

Please stay tuned for more Filecoin grant RFPs. :)

**Pull request closed without merging**